src/main/scala/com/wavesplatform/settings/WalletSettings.scala

```diff
@@ -2,6 +2,59 @@ package com.wavesplatform.settings
 
 import java.io.File
 
+import com.typesafe.config.Config
 import com.wavesplatform.state.ByteStr
+import net.ceedubs.ficus.Ficus._
 
 case class WalletSettings(file: Option[File], password: String, seed: Option[ByteStr])
+
+object WalletSettings {
+  private val configPath: String = "waves.wallet"
+
+  def fromConfig(config: Config): WalletSettings = {
+    val password: String =
+      config
+        .getAs[String](s"$configPath.password")
+        .getOrElse(readPasswordFromCommandLine())
+
+    val walletFile =
+      config
+        .as[Option[File]](s"$configPath.file")
+    val seed =
+      config
+        .as[Option[ByteStr]](s"$configPath.seed")
+
+    WalletSettings(walletFile, password, seed)
+  }
+  def readPasswordFromCommandLine(): String = {
```

Comment:
Function readPasswordFromCommandLine read a password from console but not from command line. Should we rename it?